fedramp-integrated-inventory-workbook/deployment/inventory/reports.py

```diff
@@ -0,0 +1,86 @@
+# (c) 2019 Amazon Web Services, Inc. or its affiliates. All Rights Reserved.
```

Comment:
This too, hmm

Reply:
I kept a large portion of the original repo's 'extras' that aren't directly required for the lambda. If we don't think they're helpful as resources, and list a bunch of conflicting licenses, I can trim down as much as possible?